### PR TITLE
feat: setColumHeaders

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: cypress/integration/column.spec.ts [hunk 1]
```diff
@@ -95,3 +95,40 @@ describe('setHeader()', () => {
     cy.get('[data-column-name=mergeColumn1]').should('have.text', 'information');
   });
 });
+
+describe.only('setColumnTitles()', () => {
+  it('change column titles', () => {
```
Comment (js87zz) [inline cypress/integration/column.spec.ts]: `only`는 제거해도 될 듯합니다.

File: src/dispatch/data.ts [hunk 2]
```diff
@@ -404,3 +405,23 @@ export function setPagination({ data }: Store, pageOptions: PageOptions) {
   const { perPage } = data.pageOptions;
   data.pageOptions = { ...pageOptions, perPage };
 }
+
+export function changeColumnTitlesByName({ column }: Store, columnsMap: Dictionary<string>) {
+  const { allColumns, complexHeaderColumns } = column;
+
+  Object.keys(columnsMap).forEach((key) => {
+    const col = findProp('name', key, allColumns);
```
Comment (seonim-ryu) [inline src/dispatch/data.ts]: `allColumnMap[key].name`으로 찾을 수 있을 것 같은데.. 안되나요 ㄷㄷ

File: src/dispatch/data.ts [hunk 3]
```diff
@@ -404,3 +405,23 @@ export function setPagination({ data }: Store, pageOptions: PageOptions) {
   const { perPage } = data.pageOptions;
   data.pageOptions = { ...pageOptions, perPage };
 }
+
+export function changeColumnHeadersByName({ column }: Store, columnsMap: Dictionary<string>) {
+  const { allColumns, complexHeaderColumns } = column;
+
+  Object.keys(columnsMap).forEach((key) => {
+    const col = findProp('name', key, allColumns);
```
Comment (js87zz) [inline src/dispatch/data.ts]: 파라미터 명을 `key`보단 `columnName`으로 바꾸는게 좋을 것 같고, `findProp('name', key, allColumns);` 은`allColumnMap[columnName]`으로 사용할 수 있을 듯합니다.

File: src/dispatch/data.ts (hunk 2 above)
Comment (jung-han) [reply src/dispatch/data.ts]: 아 그러네요,.....................................수정하였습니다!

**Comment by js87zz** [comment]
리뷰 완료입니다! 고생하셨습니다!

File: cypress/integration/column.spec.ts (hunk 1 above)
Comment (jung-han) [reply cypress/integration/column.spec.ts]: 제거하였습니다!

File: src/dispatch/data.ts (hunk 3 above)
Comment (jung-han) [reply src/dispatch/data.ts]: 수정하였습니다!

**Comment by seonim-ryu** [comment]
[6/13] 리뷰 완료하였습니다! 고생하셨어요ㅎ

**Pull request merged**